src/Microsoft.DotNet.Build.Tasks/Targets/packages.targets

```diff
@@ -18,6 +18,10 @@
     <NuGetOutputDirectory>&quot;$(PackagesOutDir.TrimEnd('\'))&quot;</NuGetOutputDirectory>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <LocalPublishPropsFile Condition="'$(LocalPublishPropsFile)'==''">$(SourceDir)\localpublish.props</LocalPublishPropsFile>
```

Comment:
$(PackagesDir) is what I would use in corefx. if that is not set in coreclr we can probably just set it there to be consistent.